### PR TITLE
Update L2 developer docs

### PR DESCRIPTION
**Pull request by corwintines**
## Description
- Breakout zk rollup and optimistic rollup into separate pages
- Added Metis as an ORU
- Added disclaimer next to Optimism & Metis that fraud proofs are on their roadmaps (but not yet implemented)

**Preview deploy:**
- https://ethereumorgwebsitedev01-layer2devdocs.gtsb.io/en/developers/docs/scaling
- https://ethereumorgwebsitedev01-layer2devdocs.gtsb.io/en/developers/docs/scaling/optimistic-rollups/
- https://ethereumorgwebsitedev01-layer2devdocs.gtsb.io/en/developers/docs/scaling/zk-rollups/

### CODE REVIEW

**Review by pettinarip — COMMENTED** [review]
LGTM @corwintines

File: src/data/developer-docs-links.yaml [hunk 1]
```diff
@@ -164,8 +164,10 @@
       to: /developers/docs/scaling/
       description: docs-nav-scaling-description
       items:
-        - id: docs-nav-scaling-layer-2
```
Comment (samajammin) [inline src/data/developer-docs-links.yaml:167]: We should remove this key from the JSON files.

Comment (samajammin) [reply src/data/developer-docs-links.yaml:167]: I removed!

**Review by samajammin — APPROVED** [review]
Made some updates:
- Replaced all old link references to the now dead page (/developers/docs/scaling/layer-2-rollups)
- Added a redirect from the now dead page

@corwintines or someone - could you please give a final review to sanity check my changes? Thanks!

File: src/content/translations/id/developers/docs/scaling/layer-2-rollups/index.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ Karena komputasi adalah bagian yang lambat, mahal dari penggunaan Ethereum, roll
 
 Rollup optimistic tidak menghitung transaksi, jadi perlu ada mekanisme untuk memastikan transaksi itu sah dan tidak curang. Di sinilah bukti penipuan digunakan. Jika seseorang melihat transaksi penipuan, rollup akan mengeksekusi bukti penipuan dan menjalankan perhitungan transaksi, menggunakan data state yang tersedia. Ini berarti Anda mungkin memiliki waktu tunggu yang lebih lama untuk konfirmasi transaksi dari yang diperlukan dalam rollup ZK, karena prosesnya dapat ditentang.
 
-![Diagram yang menunjukkan apa yang terjadi ketika transaksi penipuan terjadi dalam rollup Optimistic di Ethereum](../../../../../../developers/docs/scaling/layer-2-rollups/optimistic-rollups.png)
+![Diagram yang menunjukkan apa yang terjadi ketika transaksi penipuan terjadi dalam rollup Optimistic di Ethereum](../../../../../../developers/docs/scaling/#rollupsoptimistic-rollups.png)
```
Comment (corwintines) [inline src/content/translations/id/developers/docs/scaling/layer-2-rollups/index.md:57]:
```suggestion
![Diagram yang menunjukkan apa yang terjadi ketika transaksi penipuan terjadi dalam rollup Optimistic di Ethereum](../../../../../../developers/docs/scaling/optimistic-rollups.png)
```

File: src/content/translations/zh/developers/docs/scaling/layer-2-rollups/index.md [hunk 3]
```diff
@@ -54,7 +54,7 @@ Optimistic rollup 与第二层的以太坊主链同步进行。 它们可以提
 
 Optimistic rollup 并不实际计算交易，所以需要有一个机制来确保交易是合法的，而不是欺诈性的。 这就是欺诈证明的作用。 如果有人注意到有欺诈性交易，rollup 将执行防欺诈并运行交易的计算，使用可用的状态数据。 这意味着您的交易确认等待时间可能比 ZK Rollup 更长，因为它可能受到质疑。
 
-![显示以太坊中的 Optimistic rollup 中发生欺诈性交易时的图表](../../../../../developers/docs/scaling/layer-2-rollups/optimistic-rollups.png)
+![显示以太坊中的 Optimistic rollup 中发生欺诈性交易时的图表](../../../../../developers/docs/scaling/#rollupsoptimistic-rollups.png)
```
Comment (corwintines) [inline src/content/translations/zh/developers/docs/scaling/layer-2-rollups/index.md:57]:
```suggestion
![显示以太坊中的 Optimistic rollup 中发生欺诈性交易时的图表](../../../../../developers/docs/scaling/optimistic-rollups.png)
```

File: src/content/translations/pl/developers/docs/scaling/layer-2-rollups/index.md [hunk 4]
```diff
@@ -83,7 +83,7 @@ Ponieważ obliczanie to powolna, droga część korzystania z Ethereum, optymist
 
 Optymistyczne pakiety zbiorcze nie obliczają transakcji, więc potrzebny jest mechanizm zapewniający, że transakcje są legalne, a nie oszukańcze. W tym miejscu pojawiają się dowody oszustwa. Jeśli ktoś powiadomi o oszukańczej transakcji, pakiet zbiorczy wykryje oszustwo i uruchomi obliczenie transakcji przy użyciu dostępnych danych o stanie. Oznacza to, że możesz mieć dłuższy czas oczekiwania na potwierdzenie transakcji niż w przypadku pakietu zbiorczego ZK, ponieważ może być zakwestionowana.
 
-![Schemat pokazujący, co się dzieje, gdy dochodzi do oszukańczej transakcji w optymistycznym pakiecie zbiorczym w Ethereum](../../../../../../developers/docs/scaling/layer-2-rollups/optimistic-rollups.png)
+![Schemat pokazujący, co się dzieje, gdy dochodzi do oszukańczej transakcji w optymistycznym pakiecie zbiorczym w Ethereum](../../../../../../developers/docs/scaling/#rollupsoptimistic-rollups.png)
```
Comment (corwintines) [inline src/content/translations/pl/developers/docs/scaling/layer-2-rollups/index.md:86]:
```suggestion
![Schemat pokazujący, co się dzieje, gdy dochodzi do oszukańczej transakcji w optymistycznym pakiecie zbiorczym w Ethereum](../../../../../../developers/docs/scaling/optimistic-rollups/optimistic-rollups.png)
```

File: src/content/translations/ro/developers/docs/layer-2-scaling/index.md [hunk 5]
```diff
@@ -91,7 +91,7 @@ Deoarece calculul este partea lentă și costisitoare a utilizării Ethereum, ro
 
 Rollup-urile Optimistic nu calculează de fapt tranzacția, deci trebuie să existe un mecanism care să asigure că tranzacțiile sunt legitime și nu frauduloase. Aici intervin dovezile de fraudă. Dacă cineva observă o tranzacție frauduloasă, rollup-ul va executa o dovadă de fraudă și va rula calculul tranzacției, utilizând datele de stare disponibile. Aceasta înseamnă că este posibil să ai timpi de așteptare mai lungi pentru confirmarea tranzacției decât un rollup-ZK, deoarece ar putea fi contestat.
 
-![Diagramă care arată ce se întâmplă atunci când are loc o tranzacție frauduloasă într-un rollup Optimist pe Ethereum](../../../../../developers/docs/scaling/layer-2-rollups/optimistic-rollups.png)
+![Diagramă care arată ce se întâmplă atunci când are loc o tranzacție frauduloasă într-un rollup Optimist pe Ethereum](../../../../../developers/docs/scaling/#rollupsoptimistic-rollups.png)
```
Comment (corwintines) [inline src/content/translations/ro/developers/docs/layer-2-scaling/index.md:94]:
```suggestion
![Diagramă care arată ce se întâmplă atunci când are loc o tranzacție frauduloasă într-un rollup Optimist pe Ethereum](../../../../../developers/docs/scaling/optimistic-rollups/optimistic-rollups.png)
```

File: src/content/developers/docs/scaling/zk-rollups/index.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
+---
+title: Zero-Knowledge Rollups
+description: Hello World
```
Comment (corwintines) [inline src/content/developers/docs/scaling/zk-rollups/index.md:3]:
```suggestion
description: Introduction to zero-knowledge rollups
```

**Comment by corwintines** [comment]
@samajammin just fixed a couple things, but I think this is good now.

**Pull request merged**